### DISCUSSION TITLE
I'm back

### DISCUSSION THREAD

**Discussion by zloirock** [zloirock/core-js#964] [Announcements]
Wait for the news -)

**Comment by starzou** [comment]
### God bless you

**Comment by jzxznb** [comment]
好家伙，有你的

**Comment by r3nya** [comment]
Yay!

**Comment by Seminioni** [comment]
Пили пост

**Comment by joshmanders** [comment]
Welcome back!

**Comment by cepem** [comment]
Oh no

**Comment by Destaq** [comment]
When will we hear the news?

**Comment by joshxyzhimself** [comment]
Welcome back lol 🚀 🚀 🚀

**Comment by exposir** [comment]
不愧是你！

**Comment by Eve-Sama** [comment]
cn: 好家伙, 我直接好家伙!
en: Awsome, really awsome!

**Reply by Rudxain** [reply]
are you a bot?

**Reply by Rudxain** [reply]
No offense intended. I just saw A LOT of users posting the exact same thing. Is it a meme or something?

Memes usually aren't that repetitive, they should have a little bit of variation

**Reply by zaoying** [reply]
Relax dude! 
That is really a meme!
Why people likes to repeat it ? 
It's joke about "Brain in a vat". 
By encouraging people posting the same thing, it fools someone to believe that the whole internet is fake! 
People they saw online is basically the same person but with different account!

**Reply by zaoying** [reply]
> "Huh, you think you  are quite populat in the internet ? "
> "Let me tell you the truth!"
> "People you interacts online, they are identical, which is me!"
> "I create a dozen accounts, then posting thing on the internet, which makes you think there are a lot people online"
> "You want evidences? check below comments, you will see dozens of repeated message!"

**Reply by Rudxain** [reply]
@zaoying Thank you for explaining! I feel stupid now.

Your last comment (the quote) reminds me of the "It was me, Barry" meme. Both are funny!

**Comment by liwangyong** [comment]
cn: 好家伙, 我直接好家伙!
en: Awsome, really awsome!

**Comment by xuxuxu-ni** [comment]
cn: 好家伙, 我直接好家伙!
en: Awsome, really awsome!

**Comment by IAALAI** [comment]
cn: 好家伙, 我直接好家伙!
en: Awsome, really awsome!

**Comment by Ryan2128** [comment]
cn: 好家伙, 我直接好家伙!
en: Awsome, really awsome!

**Comment by jehy** [comment]
Ещё нужна работа? Пиши : )

**Comment by ELI7VH** [comment]
welcome back. you still looking for a job? haha

**Comment by liuzi6612** [comment]
🚀🚀🚀

**Comment by phugodness** [comment]
where have you been? 🤔

**Reply by elijah629** [reply]
Prison

**Reply by Rudxain** [reply]
> Prison

I didn't believe this, so I put a "laugh reaction". But now I know this is real. I hate ~Russia~ the Russian Government, almost as much as Apple

**Comment by ghost** [comment]
Welcome back!

**Comment by OtacilioN** [comment]
> where have you been? 🤔

In jail

**Reply by Rudxain** [reply]
wait, really?!

**Reply by Rudxain** [reply]
oh wait, it is true... damn... I hate Russia

Edit: I mean I hate the Russian government, not Russian people

**Comment by skyplaying** [comment]
大佬出来就有好工作了

**Comment by SagarBajpai** [comment]
Welcome bro! <3

**Comment by shengyan-bit** [comment]
You still looking for a job?

**Comment by nabilfreeman** [comment]
Just checked in on you after a while - welcome back and thanks so much for your OSS contributions!!! 🌅

**Comment by Rudxain** [comment]
@zloirock Bro, you should get some money and take a full vacation, **for a year**. Ask @tc39 if they want to host and maintain this repo. Perhaps even the team at @Mozilla

**Comment by junaga** [comment]
thank you..
I am so sorry, about everything.

I wish you the best.

**Comment by vincebanzon** [comment]
Thank you for your hard work and dedication. Surely you inspired a lot of people.
